vim25/soap/client.go

```diff
@@ -45,14 +45,19 @@ type RoundTripper interface {
 	RoundTrip(ctx context.Context, req, res HasFault) error
 }
 
+var DefaultVimNamespace = "urn:vim25"
+var DefaultVimVersion = "6.0"
+
 type Client struct {
 	http.Client
 
-	u *url.URL
-	k bool // Named after curl's -k flag
-	d *debugContainer
-	t *http.Transport
-	p *url.URL
+	u   *url.URL
+	k   bool // Named after curl's -k flag
+	d   *debugContainer
+	t   *http.Transport
+	p   *url.URL
+	Ns  string // Vim namespace
```

Comment:
Let's name these fields 'VimNamespace' and 'VimVersion'.  Also put an empty line before the VimNamespace field, this will maintain the alignment of the private fields.